kivy/utils.py

```diff
@@ -110,17 +110,34 @@ def rgba(s, *args):
     raise Exception('Invalid value (not a string / list / tuple)')
 
 
-def get_color_from_hex(s):
+def get_color_from_hex(color):
     '''Transform a hex string color to a kivy
     :class:`~kivy.graphics.Color`.
+
+    .. versionchanged:: 2.2.0
+
+    Supporting and 3-4 HEX values.
```

Comment:
How about "Added support for 3-character and 4-character HEX RGB shorthands" ?

Reply:
English is not my first language, so yeah, sorry for grammar.

How about `Added support for 3-character and 4-character HEX RGB(A) shorthands`?